src/records/Project.js

```diff
@@ -17,7 +17,7 @@ export default class Project extends Record({
   sources: new Sources(),
   enabledLibraries: new Set(),
   hiddenUIComponents: new Set(['console']),
-  updatedAt: null,
+  updatedAt: new Date(),
```

Comment:
@jwang1919 this is a good thought, but unfortunately it has far-reaching consequences that I don’t think we want. In particular, the concept of a “pristine project”, namely one that does not have an `updatedAt` set, is used to make decisions in several areas of the codebase, including:

* Whether to persist a project to Firebase in various situations
* Whether to keep a project listed in the **My Projects** menu after you switch away from it

This change inadvertently removes the whole “pristine” concept from the application by making it a state that a project can never be in.

The most obvious undesirable effect can be observed by checking out this branch and repeatedly pressing **New Project**: you’ll note that your projects list fills up with each new project you create. Whereas in the current released version of Popcode, a new project will only stick around in the menu if you modify it before switching away from it.

So, instead of making it so that projects are never pristine, I think we just want to change the logic specific to the **My Projects** menu, telling it to include the current project in the list even if the current project is pristine.